docs/content/drafts/PageLayout.mdx

```diff
@@ -82,7 +148,12 @@ description: TODO
 ### PageLayout.Header
 
 <PropsTable>
-  <PropsTableRow name="divider" type="'none' | 'line'" defaultValue="'none'" />
+  <PropsTableRow
+    name="divider"
+    type={`| 'none'
+| 'line'`}
+    defaultValue="'none'"
```

Comment:
if `line` is the only real alternative, should this be a boolean instead like `showDivider`?

Reply:
Just noticed `horizontalDividerVariants`. Is this missing 'filled'?

Reply:
Yeah, `filled` is only an option at narrow viewports. I chose `divider: none | line` instead of `showDivider: boolean` to align with the `dividerWhenNarrow` prop.